specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/GetModel.json

```diff
@@ -32,8 +32,19 @@
           "fields": [],
           "averageModelAccuracy": 0,
           "errors": []
-        }
+        },
+        "composedTrainResults": [
+          {
+          "trainResult": {
+            "trainingDocuments": [],
```

Comment:
Would it make sense to add a property like formTypeId to parallel clusterId in KeysResult?